mock-echo-app/src/cmr/mock_echo/api/urs.clj

```diff
@@ -109,8 +111,10 @@
       successful-login-response
       unsuccessful-login-response)))
 
+(def bees (byte-array 10))
```

Comment:
What is this? Is it used anywhere?

Reply:
Who doesn't need bees? Looks like some of my debugging didn't get deleted.

Reply:
![image](https://media1.tenor.com/images/41f36d9db4b706749cf96909b2b0a133/tenor.gif)